src/features/Footer/Footer.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { ReactComponent as Logo } from '../../assets/icons/footer-logo.svg';
+import FooterItem from './components/FooterItem';
+
+const Footer = () => {
+  const scrollToTop = () => window.scrollTo({ top: 0 });
+
+  return (
+    <div className="h-44 mt-24 flex items-center bg-secondary">
```

Comment:
I think it's better to provide margins (mt-24) as passed by props or wrappers for components. 
Now footer has forced margin which makes it less reusable